PKGBUILDS/phosh/danctnix-phosh-ui-meta/PKGBUILD

```
@@ -19,5 +19,6 @@ depends=(
 	sound-theme-librem5
 	iio-sensor-proxy
 	wys
+	xdg-desktop-portal-gnome
```

xdg-desktop-portal-gnome will slow down GNOME apps.

```suggestion
	xdg-desktop-portal-gtk
```

---

Thanks, looks `xdg-desktop-portal-gtk` also fixes the issue on it's own. I updated the PR.